sympy/combinatorics/homomorphisms.py

```diff
@@ -487,21 +487,28 @@ def group_isomorphism(G, H):
 
         if not counterexample:
             # Returns a `GroupHomomorphism` object.
-            return homomorphism(G, H, G.generators, H, check=False)
+            return homomorphism(G, H, G.generators, subset, check=False)
 
     return None
 
-def is_isomorphic(G, H):
+def is_isomorphic(G, H, isomorphism=False):
     '''
     Check if the given groups are isomorphic.
 
     Arguments:
         G (a finite `FpGroup` or a `PermutationGroup`) -- First group
         H (a finite `FpGroup` or a `PermutationGroup`) -- Second group
+        isomorphism(boolean) -- This computes the isomorphism when set to True
```

Comment:
`This returns an isomorphism...` is possibly better because isomorphisms aren't unique and it's computed either way.